announcement/models/announcement.py

```diff
@@ -95,8 +98,7 @@ def _onchange_announcement_type(self):
             self.specific_user_ids = False
 
     def action_announcement_log(self):
-        """Return a set of fungible transient records to see altogether read logs
-        and pendant users"""
+        """See altogether read logs and pendant users"""
```

Comment:
```suggestion
        """See altogether read logs and pending users"""
```

Reply:
Fixed changin it to *unread* as @gdgellatly suggested